engines/mm/xeen/worldofxeen/worldofxeen_menu.cpp

```diff
@@ -527,39 +599,44 @@ void OtherOptionsDialog::loadButtons() {
 	Common::Rect r(93, 53, 227, 73);
 
 	// View Darkside Intro
-	addButton(r, Common::KEYCODE_d, &_buttonSprites);
+	addButton(r, Common::KEYCODE_d, &_buttonSprites, 0);
 	r.translate(0, 25);
 
 	// View Clouds Intro
 	if (g_vm->getGameID() == GType_WorldOfXeen) {
-		addButton(r, Common::KEYCODE_c, &_buttonSprites);
+		addButton(r, Common::KEYCODE_c, &_buttonSprites, 1);
 		r.translate(0, 25);
 	} else {
-		addButton(Common::Rect(), Common::KEYCODE_INVALID);
+		addButton(Common::Rect(), Common::KEYCODE_INVALID, nullptr, 1);
```

Comment:
I am not a huge fan of magic numbers like this. Is there a way and sense in general, of putting these indexes to an enum?